lib/AST/DistributedDecl.cpp

```diff
@@ -1333,6 +1333,20 @@ AbstractFunctionDecl *ASTContext::getRemoteCallOnDistributedActorSystem(
 /********************** Distributed Actor Properties **************************/
 /******************************************************************************/
 
+FuncDecl *VarDecl::getDistributedThunk() const {
+  if (!isDistributed())
+    return nullptr;
+
+  // Only get-only 'distributed' computed properties are considered valid.
+  if (isStatic() || isLet() || hasStorageOrWrapsStorage() ||
+      getWriteImpl() != swift::WriteImplKind::Immutable)
+    return nullptr;
```

Comment:
We can remove this since https://github.com/apple/swift/pull/59481/commits/4cf402bd031b8b82cc1bd6f61aae8dd97cf06b88 fixes the checking to bail out when the prop is invalid, so we don't need to replicate the logic 👍

Reply:
Sounds good!